lib/licenses.nix

```diff
@@ -1212,6 +1212,12 @@ lib.mapAttrs mkLicense ({
     fullName = "TCL/TK License";
   };
 
+  tost = {
+    shortName = "TSL";
+    fullName = "Tomorrow Open Source License";
+    url = "https://github.com/PixarAnimationStudios/OpenUSD/blob/release/LICENSE.txt";
```

Comment:
If this is the only project which uses this license (as evidenced by the fact that it's only available in their source repo), I think it'd be better to just have it as a custom license in `meta.license` without adding it to `lib.licenses`.

Reply:
> If this is the only project which uses this license (as evidenced by the fact that it's only available in their source repo), I think it'd be better to just have it as a custom license in `meta.license` without adding it to `lib.licenses`.

TBF they could use it for future projects as well (Pixar, I mean), but yeah.

Reply:
Good to know, in that case it should be added and used in both packages.